extract_dmd.d

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env dub
+/++
+dub.sdl:
+dependency "libdparse" version="0.7.0-beta.7"
+dependency "mustache-d" version="0.1.3"
```

Comment:
There are three variables defined and used in the Mustache context (1 in `grammar.d` and 2 in `traits.d`, so `replace` would look a bit uglier, but yes it would definitely work as well.

Btw mustache-d is one of the [20 libs we test on Jenkins](https://ci.dlang.io/blue/organizations/jenkins/dlang-org%2Fphobos/detail/master/137/pipeline).

So I am happy to change this if we decide to go this road.